Gruntfile.js

```diff
@@ -37,7 +37,7 @@ module.exports = (grunt) => {
         library: libraryName,
         libraryTarget: 'umd',
         libraryExport: 'default',
-        globalObject: `typeof window !== 'undefined' ? window : this`,
+        globalObject: `typeof window !== 'undefined' ? window : typeof globalThis !== 'undefined' ? globalThis : this`,
```

Comment:
@0xfe I have tested this with legacy Node JS which does not include globalThis. Guarding it with typeof prevents any errors, and enables VexFlow to be imported as a JS module in Node.

Reply:
Can you add a comment here (with a bit of detail) around what's going on here?